app/src/main/java/org/isoron/uhabits/models/ScoreList.java

```diff
@@ -89,6 +89,24 @@ public final int getValue(long timestamp)
         return s.getValue();
     }
 
+    /**
+     * Gets the values of all the scores between the given days.
+     *
+     * @param from the timestamp of the first day
+     * @param to the timestamp of the second day
+     * @return the values of all the scores between the given days
+     */
+    public final int[] getValues(long from, long to)
```

Comment:
Woops, sorry about that!

I've fixed the issue and added the test case as well as test cases for the other functions I've added.
I also changed the newest date to use when exporting the csv to the current day, since all the other files use the current day (but I've left getTimeFrame in, in case this needs to be used by another function in the future).